docs/architecture.md

```diff
@@ -191,23 +191,31 @@ so their source IP will be rewritten to the Node's IP before going out.
 
 ### ClusterIP Service
 
-<img src="/docs/assets/service_walk.svg.png" width="600" alt="Antrea Service Traffic Walk">
+Antrea supports two ways to implement Services of type ClusterIP - leveraging
+`kube-proxy`; or AntreaProxy that implements load balancing for ClusterIP
```

Comment:
why is there a semi-colon here?

Reply:
I wanted to separate the two options. Let me know if you think ";" should be removed.

Reply:
I think a comma is what's appropriate here

or a markdown list

Reply:
Changed to ",".